bin/start_backend.sh

```diff
@@ -171,9 +171,18 @@ fi
 
 if [ ${RUN_BE} -eq 1 ]; then
     pidfile=$PID_DIR/be.pid
+    process_name="backend"
+    conf_file="$STARROCKS_HOME/conf/be.conf"
 fi
 if [ ${RUN_CN} -eq 1 ]; then
     pidfile=$PID_DIR/cn.pid
+    process_name="compute node"
+    conf_file="$STARROCKS_HOME/conf/cn.conf"
+fi
+
+http_port=$(grep -v ^# $conf_file |grep be_http_port| sed 's/.*= *//')
```

Comment:
this already loaded by `export_env_from_conf $STARROCKS_HOME/conf/cn.conf` in line 72